caluma-demo/app/templates/documents/edit.hbs

```diff
@@ -1,3 +1,4 @@
 {{#if model}}
   {{cf-content documentId=model}}
 {{/if}}
+{{outlet}}
```

Comment:
Since you don't have a subroute here, delete the outlet

Reply:
Ah right, I've deleted it.